### PR TITLE
chore: create tsconfig.json file

### PR DESCRIPTION
**Pull request by wgola**
Created tsconfig.json file and added it to project

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
+    "compilerOptions": {
+      "allowUnreachableCode": false,
+      "noFallthroughCasesInSwitch": true,
+      "noImplicitReturns": true,
+      "strict": true,
+      "noUnusedLocals": true,
+      "noUnusedParameters": true,
+
```
Comment (jtowarnicka) [inline tsconfig.json:9]: usun puste linie

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+      "noImplicitReturns": true,
+      "strict": true,
+      "noUnusedLocals": true,
+      "noUnusedParameters": true,
+  
+      "baseUrl": ".",
+      "module": "CommonJS",
+      "moduleResolution": "node",
+      "resolveJsonModule": true,
+
```
Comment (jtowarnicka) [inline tsconfig.json:14]: to samo

File: tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+      "noUnusedLocals": true,
+      "noUnusedParameters": true,
+  
+      "baseUrl": ".",
+      "module": "CommonJS",
+      "moduleResolution": "node",
+      "resolveJsonModule": true,
+  
+      "noEmit": true,
+
```
Comment (jtowarnicka) [inline tsconfig.json:16]: tez

File: tsconfig.json [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+      "baseUrl": ".",
+      "module": "CommonJS",
+      "moduleResolution": "node",
+      "resolveJsonModule": true,
+  
+      "noEmit": true,
+      
+      "allowJs": true,
+      "checkJs": true,
+
```
Comment (jtowarnicka) [inline tsconfig.json:19]: ^

File: tsconfig.json [hunk 5]
```diff
@@ -0,0 +1,33 @@
...
+      "noEmit": true,
+      
+      "allowJs": true,
+      "checkJs": true,
+  
+      "allowSyntheticDefaultImports": true,
+      "forceConsistentCasingInFileNames": true,
+      "isolatedModules": true,
+      "esModuleInterop": true,
+
```
Comment (jtowarnicka) [inline tsconfig.json:24]: ^

File: tsconfig.json [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+      "allowJs": true,
+      "checkJs": true,
+  
+      "allowSyntheticDefaultImports": true,
+      "forceConsistentCasingInFileNames": true,
+      "isolatedModules": true,
+      "esModuleInterop": true,
+      
+      "jsx": "react-jsx",
+
```
Comment (jtowarnicka) [inline tsconfig.json:26]: ^

**Review by jtowarnicka — APPROVED** [review]
LGTM

File: tsconfig.json (hunk 1 above)
Comment (wgola) [reply tsconfig.json]: Resolved

File: tsconfig.json (hunk 2 above)
Comment (wgola) [reply tsconfig.json]: Resolved

File: tsconfig.json (hunk 3 above)
Comment (wgola) [reply tsconfig.json]: Resolved

File: tsconfig.json (hunk 4 above)
Comment (wgola) [reply tsconfig.json]: Resolved

File: tsconfig.json (hunk 5 above)
Comment (wgola) [reply tsconfig.json]: Resolved

File: tsconfig.json (hunk 6 above)
Comment (wgola) [reply tsconfig.json]: Resolved

**Pull request merged**